services/harmony/test/helpers/ogc-api-edr.ts

```diff
@@ -71,19 +71,21 @@ export function hookLandingPage(collection: string, version: string): void {
  * @param app - The express application (typically this.frontend)
  * @param version - The OGC EDR API version
  * @param collection - The CMR Collection ID to perform a service on
+ * @param queryType - the type of call to make, e.g., 'cube'
  * @param options - additional options for the request
  * @returns The response
  */
 export function edrRequest(
   app: Application,
   version: string = defaultVersion,
   collection: string = defaultCollection,
+  queryType: 'cube' | 'area' = 'cube',
```

Comment:
I think longer term we should drop the default and update any of the cube tests to pass this in.

Reply:
Done.